pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```diff
@@ -47,13 +50,15 @@ def __init__(
         assembly: str,
         github_token: str,
         gitlab_token: str,
+        kubeconfig: Optional[str] = None,
         build_repo_url: Optional[str] = None,
         shipment_repo_url: Optional[str] = None,
         job_url: Optional[str] = None,
     ) -> None:
         self.runtime = runtime
         self.assembly = assembly
         self.group = group
+        self.kubeconfig = kubeconfig
```

Comment:
We have a pattern of reading the specifc env var "KONFLUX_SA_KUBECONFIG" as the kubeconfig var , so instead of creating this param, can you make prepare-release or doozer bundle cmd automatically rely on this?